src/components/RightSidebar/PinnedMessages/PinnedMessageItem.vue

```diff
@@ -0,0 +1,136 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<script setup lang="ts">
+import type { PinnedChatMessage } from '../../../types/index.ts'
+
+import { t } from '@nextcloud/l10n'
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+import { useStore } from 'vuex'
+import NcActionButton from '@nextcloud/vue/components/NcActionButton'
+import NcListItem from '@nextcloud/vue/components/NcListItem'
+import IconPinOff from 'vue-material-design-icons/PinOffOutline.vue'
+import IconPin from 'vue-material-design-icons/PinOutline.vue'
+import { useGetToken } from '../../../composables/useGetToken.ts'
+import { AVATAR } from '../../../constants.ts'
+import { EventBus } from '../../../services/EventBus.ts'
+import { useActorStore } from '../../../stores/actor.ts'
+import { useSharedItemsStore } from '../../../stores/sharedItems.ts'
+import { formatDateTime } from '../../../utils/formattedTime.ts'
+import { parseToSimpleMessage } from '../../../utils/textParse.ts'
+
+const { item } = defineProps<{ item: PinnedChatMessage }>()
+const store = useStore()
+const token = useGetToken()
+const route = useRoute()
+const actorStore = useActorStore()
+const sharedItemsStore = useSharedItemsStore()
+const isModerator = computed(() => store.getters.isModerator)
+
+const timeCaption = computed(() => {
+	if (!item.metaData.pinnedUntil) {
+		const isPinnerSameAsCurrentUser = actorStore.checkIfSelfIsActor({
+			actorType: item.metaData.pinnedActorType,
+			actorId: item.metaData.pinnedActorId,
+		})
+		if (isPinnerSameAsCurrentUser) {
+			return t('spreed', 'Pinned by you')
+		} else if (item.metaData.pinnedActorDisplayName) {
+			return t('spreed', 'Pinned by {name}', {
```

Comment:
as everywhere else for users
```suggestion
			return t('spreed', 'Pinned by {actor}', {
```